l3build/l3build.lua

```diff
@@ -565,27 +565,13 @@ function mkdir(dir)
   end
 end
 
--- Find the relationship between two directories
-function relpath(target, source)
-  -- A shortcut for the case where the two are the same
-  if target == source then
-    return ""
-  end
-  local resultdir = ""
-  local trimpattern = "^[^/]*/"
-  -- Trim off identical leading directories
-  while
-    (match(target, trimpattern) or "X") ==
-    (match(target, trimpattern) or "Y") do
-    target = gsub(target, trimpattern, "")
-    source = gsub(source, trimpattern, "")
-  end
-  -- Go up from the source
-  for i = 0, select(2, gsub(target, "/", "")) do
-    resultdir = resultdir .. "../"
-  end
-  -- Return the relative part plus the unique part of the target
-  return resultdir .. target
+-- Return an absolute path from a relative one
```

Comment:
I guess what I'm trying to get at is that provided the two locations are on the same disk there is a relative path from one to the other, so one should be able to work out the way to traverse it.